.travis.yml

```diff
@@ -6,5 +6,6 @@ node_js:
   - "0.10"
 after_script:
   - npm run coveralls
+  - npm run lint
```

Comment:
I don't think `after_script` fails the build if something errors in it.  Also, I think we should switch from iojs to node 4 in testing.